data/xml/2015.mtsummit.xml

```diff
@@ -0,0 +1,525 @@
+      <author><first>Achim</first><last>Ruopp</last></author>
+      <url hash="11c11366">2015.mtsummit-users.14</url>
+      <bibkey>ruopp-2015-industry</bibkey>
+    </paper>
+    <paper id="15">
+      <title>Accurately predicting post-editing time &amp; labor for cost management</title>
+      <author><first>Carla</first><last>Schelfhout</last></author>
+      <url hash="58cf3cbc">2015.mtsummit-users.15</url>
+      <bibkey>schelfhout-2015-accurately</bibkey>
+    </paper>
+    <paper id="16">
+      <title>Beyond rext, machine translation and <fixed-case>NLP</fixed-case> for e-discovery [abstract]</title>
+      <author><first>Jean</first><last>Sennellart</last></author>
+      <author><first>Denis</first><last>Gachot</last></author>
+      <author><first>Joshua</first><last>Johanson</last></author>
+      <url hash="a94f6705">2015.mtsummit-users.16</url>
+      <bibkey>sennellart-etal-2015-beyond</bibkey>
+    </paper>
+    <paper id="17">
+      <title>Designing user experience for machine translated conversations</title>
+      <author><first>Tanvi</first><last>Surti</last></author>
+      <url hash="891c347f">2015.mtsummit-users.17</url>
+      <bibkey>surti-2015-designing</bibkey>
+    </paper>
+    <paper id="18">
+      <title>How much cake is enough: the case for domain-specific engines</title>
+      <author><first>Alex</first><last>Yanishevsky</last></author>
+      <url hash="7ec00285">2015.mtsummit-users.18</url>
+      <bibkey>yanishevsky-2015-much</bibkey>
+    </paper>
+    <paper id="19">
+      <title>Productivity promotion strategies for collaborative translation on huge volume technical documents</title>
+      <author><first>Guiping</first><last>Zhang</last></author>
+      <author><first>Na</first><last>Ye</last></author>
+      <author><first>Fang</first><last>Cai</last></author>
+      <author><first>Chuang</first><last>Wu</last></author>
+      <author><first>Xiangkui</first><last>Sun</last></author>
+      <author><first>Jinfu</first><last>Yuan</last></author>
+      <author><first>Dongfeng</first><last>Cai</last></author>
+      <url hash="e75bb828">2015.mtsummit-users.19</url>
+      <bibkey>zhang-etal-2015-productivity</bibkey>
+    </paper>
+    <paper id="20">
+      <title>A machine-assisted human translation system for technical documents</title>
+      <author><first>Vishwajeet</first><last>Kumar</last></author>
+      <author><first>Ashish</first><last>Kulkarni</last></author>
+      <author><first>Pankaj</first><last>Singh</last></author>
+      <author><first>Ganesh</first><last>Ramakrishnan</last></author>
+      <author><first>Ganesh</first><last>Arnaal</last></author>
+      <url hash="95c4f737">2015.mtsummit-users.20</url>
+      <bibkey>kumar-etal-2015-machine</bibkey>
+    </paper>
+  </volume>
+  <volume id="wptp" ingest-date="2021-08-03">
+    <meta>
+      <booktitle>Workshop on Post-editing Technology and Practice</booktitle>
```

Comment:
This should be “Proceedings of the 4th Workshop on Post-editing Technology and Practice”